config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties

```diff
@@ -5492,6 +5492,9 @@
 # The email address that will receive the confirmation mails from the import jobs (by default uses the defined in portal.error.email)
 #tags.import_job_email=youremail@yourdomain.ext
 
+# Allow the use of tags in some tools: assignments, private messages (default is false)
+# tagservice.enable.integrations=true
```

Comment:
It must be enabled by default, put S2U-32 as prefix for the feature and state that the default is ENABLED